### PR TITLE
Brocha E commit

### PR DESCRIPTION
**Pull request by Brochae**
Testing

### CODE REVIEW

File: 0 database.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+use master
+go
+drop database if exists Universe
```
Comment (mordechaigellis) [inline 0 database.sql:3]: make it lower case

**Review by mordechaigellis — CHANGES_REQUESTED** [review]

File: 0 database.sql (hunk 1 above)
Comment (Brochae) [reply 0 database.sql]: changed it to lowercase please review

File: 0 database.sql [hunk 2]
```diff
@@ -1,8 +1,8 @@
 use master
 go
-drop database if exists Universe
+drop database if exists universeDB
 go
-create database Universe
+create database universeDB
 go
-use Universe
+use universeDB
```
Comment (mordechaigellis) [inline 0 database.sql:7]: DB is still uppercase, please correct

Comment (Brochae) [reply 0 database.sql]: corrected

File: 0 database.sql [hunk 3]
```diff
@@ -0,0 +1,8 @@
+use master
+go
+drop database if exists universedb
+go
+create database universedb
+go
+use universedb
+go
```
Comment (mordechaigellis) [inline 0 database.sql:8]: you still need to do the table

**Pull request closed without merging**